### PR TITLE
Switched to  java 11

### PR DESCRIPTION
**Pull request by defrances**
Dockerfile was updated

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -12,11 +12,20 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
+
                 <configuration>
                     <source>1.8</source>
                     <target>1.8</target>
                 </configuration>
             </plugin>
+            <plugin>
```
Comment (extsoft) [inline pom.xml]: @defrances Why do we ignore failures of unit tests?

File: pom.xml [hunk 2]
```diff
@@ -62,5 +71,15 @@
             <artifactId>allure-testng</artifactId>
             <version>2.0-BETA3</version>
         </dependency>
+        <dependency>
```
Comment (extsoft) [inline pom.xml]: @defrances Why do we need this dependency?

File: pom.xml [hunk 3]
```diff
@@ -62,5 +71,15 @@
...
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-simple</artifactId>
+            <version>1.6.1</version>
+        </dependency>
+        <dependency>
```
Comment (extsoft) [inline pom.xml]: @defrances Why do we need this dependency?

File: jcat.sh [hunk 4]
```diff
@@ -1,6 +1,5 @@
 #!/bin/sh
 set -e
-
 COMMAND=$(echo "java -Dselenium-url=${SELENIUM_URL} -Dbrowser=${BROWSER} -jar jcat.jar")
 echo "Execute: ${COMMAND}"
-eval "${COMMAND}"
+eval "${COMMAND}"
```
Comment (extsoft) [inline jcat.sh]: @defrances why did you update this file?

File: pom.xml [hunk 5]
```diff
@@ -12,11 +12,20 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
+
                 <configuration>
                     <source>1.8</source>
```
Comment (extsoft) [inline pom.xml]: @defrances is it really Java 11?

File: pom.xml (hunk 3 above)
Comment (defrances) [reply pom.xml]: @extsoft When I try run build new container I faced with this issue in my logs SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
This dependency can resolve this issue

File: jcat.sh (hunk 4 above)
Comment (defrances) [reply jcat.sh]: @extsoft I faced with some issue when I try push this file in docker container
- When I pull original code from git this file has line separator "CRLF" (seems it's happened because I am use Windows ). For fix this issue I change line seporator "LF"
- When I try run this script on Linux  I faced with empty lines issue

File: pom.xml (hunk 5 above)
Comment (defrances) [reply pom.xml]: @extsoft seems like yes could you please take a look in attached screen

File: pom.xml (hunk 3 above)
Comment (extsoft) [reply pom.xml]: I see.

File: jcat.sh (hunk 4 above)
Comment (extsoft) [reply jcat.sh]: @defrances please configure Git to checkout unix-style line endings. And then, you don't need to update this file. And you have to revet it.

File: pom.xml (hunk 5 above)
Comment (extsoft) [reply pom.xml]: @defrances the screenshot just shows that you use Java 11 for compiling the project. But the sources and outputs (bytecode) are in Java 8 (see https://maven.apache.org/plugins/maven-compiler-plugin/examples/set-compiler-source-and-target.html for the details). You have to update the configuration.

Comment (defrances) [reply pom.xml]: @extsoft Updated

File: jcat.sh (hunk 4 above)
Comment (defrances) [reply jcat.sh]: @extsoft Updated

File: pom.xml (hunk 1 above)
Comment (extsoft) [reply pom.xml]: @defrances this question is still open.

File: jcat.sh (hunk 4 above)
Comment (extsoft) [reply jcat.sh]: @defrances good job! Resolve.

File: pom.xml (hunk 5 above)
Comment (extsoft) [reply pom.xml]: @defrances good job! Resolved.

File: pom.xml (hunk 1 above)
Comment (defrances) [reply pom.xml]: @extsoft Added small fix for run with selenium grid

File: src/main/java/pro/extsoft/comments/Base.java [hunk 6]
```diff
@@ -36,7 +36,7 @@ protected final WebDriver browser() {
     @BeforeTest
     public void setUp() throws MalformedURLException {
         final WebDriver driver = new RemoteWebDriver(
-                new URL(System.getProperty("selenium-url", "http://localhost:9515")),
+                new URL(System.getProperty("selenium-url", "http://192.168.56.1:4444/wd/hub")),
```
Comment (extsoft) [inline src/main/java/pro/extsoft/comments/Base.java]: @defrances why did you put here the IP? If you run without Docker, you have to start `chromedriver` that runs on `localhost:9515`.

Comment (defrances) [reply src/main/java/pro/extsoft/comments/Base.java]: @extsoft  Remove specific IP and left localhost

File: src/main/java/pro/extsoft/comments/Base.java [hunk 7]
```diff
@@ -36,7 +36,7 @@ protected final WebDriver browser() {
     @BeforeTest
     public void setUp() throws MalformedURLException {
         final WebDriver driver = new RemoteWebDriver(
-                new URL(System.getProperty("selenium-url", "http://localhost:9515")),
+                new URL(System.getProperty("selenium-url", "http://localhost:9515/wd/hub")),
```
Comment (extsoft) [inline src/main/java/pro/extsoft/comments/Base.java]: @defrances why did you add `/wd/hub`?

File: pom.xml [hunk 8]
```diff
@@ -12,9 +12,18 @@
...
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <version>2.19.1</version>
                 <configuration>
-                    <source>1.8</source>
-                    <target>1.8</target>
+                    <testFailureIgnore>true</testFailureIgnore>
```
Comment (extsoft) [inline pom.xml]: @defrances Why do we ignore failures of unit tests?

File: README.md [hunk 9]
```diff
@@ -7,7 +7,7 @@ This repo provides a sample set of automated tests for [http://commentssprintone
 ![](https://travis-ci.org/extsoft/jcat.svg?branch=master)
 
 ## Used tools
-- [Java 8](http://www.oracle.com/technetwork/java/javase/overview/java8-2100321.html) as a programming language
+- [Java 11](https://www.oracle.com/technetwork/java/javase/downloads/index.html) as a programming language
```
Comment (extsoft) [inline README.md]: @defrances please put the link to OpenJDK as it is free and used in Docker image - https://jdk.java.net/java-se-ri/11

Comment (defrances) [reply README.md]: @extsoft Done

File: src/main/java/pro/extsoft/comments/Base.java (hunk 7 above)
Comment (defrances) [reply src/main/java/pro/extsoft/comments/Base.java]: @extsoft it's  selenium grid specific

Comment (extsoft) [reply src/main/java/pro/extsoft/comments/Base.java]: But if you run chromedriver, you don't need `/wd/hub` prefix 
```bash
~ chromedriver 
Starting ChromeDriver 2.41.578706 (5f725d1b4f0a4acbf5259df887244095596231db) on port 9515
Only local connections are allowed.
```
Or did I missed something?

File: pom.xml (hunk 8 above)
Comment (defrances) [reply pom.xml]: @extsoft removed this configuration

File: src/main/java/pro/extsoft/comments/Base.java (hunk 7 above)
Comment (defrances) [reply src/main/java/pro/extsoft/comments/Base.java]: @extsoft Leave this link http://localhost:9515

File: pom.xml [hunk 10]
```diff
@@ -12,11 +12,17 @@
...
                 <artifactId>maven-compiler-plugin</artifactId>
+
                 <configuration>
-                    <source>1.8</source>
-                    <target>1.8</target>
+                    <source>11</source>
+                    <target>11</target>
                 </configuration>
             </plugin>
+            <plugin>
```
Comment (extsoft) [inline pom.xml]: @defrances do we really need this plugin here?
